_rules/SC1-3-1-aria-required-owned-element.md

```diff
@@ -0,0 +1,150 @@
+---
+name: ARIA required owned elements
+
+description: |
+	The rule checks existence of role(s) on required owned elements.
```

Comment:
Suggesting a rewording: The rule checks that for an explicit semantic role all required owned elements exists

Reply:
"The rule checks that a role has at least one of its required owned elements."

My understanding was that an owned element need only list one of its required owned elements as its role